package_manifest.yaml

```diff
@@ -358,46 +359,45 @@ katello_packages:
     rubygem-robotex: {}
     rubygem-runcible: {}
 
-katello_client_packages:
+foreman_client_packages:
   vars:
-    package_base_dir: 'packages/katello/'
+    package_base_dir: 'packages/client/'
     releasers:
-    - koji-katello-client
+    - koji-foreman-client
     repoclosure_config: repoclosure/yum.conf
     repoclosure_lookaside_repos:
       - el7-base
       - el7-updates
       - el7-epel
       - el7-extras
       - el7-puppet-5
-      - el7-katello-client-nightly
+      - el7-foreman-client-nightly
       - el6-base
       - el6-updates
       - el6-epel
       - el6-extras
       - el6-puppet-5
       - el6-subscription-manager
       - el6-qpid
-      - el6-katello-client-nightly
+      - el6-foreman-client-nightly
       - el5-base
       - el5-updates
       - el5-epel
       - el5-extras
       - el5-puppet-5
       - el5-subscription-manager
       - el5-qpid
-      - el5-katello-client-nightly
+      - el5-foreman-client-nightly
       - f28-base
       - f28-updates
       - f28-puppet-5
-      - f28-katello-client-nightly
+      - f28-foreman-client-nightly
       - f27-base
       - f27-updates
       - f27-puppet-5
-      - f27-katello-client-nightly
+      - f27-foreman-client-nightly
   hosts:
     katello-host-tools: {}
-    katello-repos: {}
```

Comment:
Why does this need to be removed?

Reply:
The section is being renamed to foreman client repos. I don't think it makes sense for the client tools to ship the katello repos (which should be named `katello-release` if it follows the naming convention), especially if katello-repos is now server-only.